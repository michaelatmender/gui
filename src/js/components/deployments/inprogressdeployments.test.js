import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { createMount } from '@material-ui/core/test-utils';
import Progress from './inprogressdeployments';

it('renders correctly', () => {
  const tree = createMount()(
    <MemoryRouter>
      <Progress progress={[]} />
    </MemoryRouter>
  );
  expect(tree.html()).toMatchSnapshot();
});
