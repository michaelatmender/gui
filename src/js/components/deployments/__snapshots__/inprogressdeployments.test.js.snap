// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"<div class=\\"fadeIn\\"><div class=\\"deploy-table-contain\\"><div class=\\"loaderContainer shrunk\\"><div class=\\"loader\\"><span class=\\"dot dot_1\\"></span><span class=\\"dot dot_2\\"></span><span class=\\"dot dot_3\\"></span><span class=\\"dot dot_4\\"></span></div></div><div class=\\"dashboard-placeholder\\"><p>Pending and ongoing deployments will appear here. </p><p><a>Create a deployment</a> to get started</p><img src=\\"assets/img/deployments.png\\" alt=\\"In progress\\"></div></div></div>"`;
